MAUI/ProgressBar/migration.md

```diff
@@ -0,0 +1,587 @@
+---
+layout: post
+title: Migrate from Xamarin to .NET MAUI  ProgressBar | Syncfusion
+description: Learn here all about Migrating from Syncfusion Xamarin SfProgressBar to Syncfusion .NET MAUI SfProgressBar control and more.
```

Comment:
description: Learn here all about Migrating from the Syncfusion Xamarin SfProgressBar to the Syncfusion .NET MAUI SfProgressBar control and more.

Reply:
Done